tests/test_pint_units.py

```diff
@@ -0,0 +1,97 @@
+from math import pi
+
+import pytest
+from pint import Quantity
+
+from pnpq.units import ureg
+
+
+@pytest.mark.parametrize(
+    "test_mpc320_step, expected_angle",
+    [
+        (-100, -100 * (170 / 1370)),
+        (0, 0),
+        (100, 100 * (170 / 1370)),
```

Comment:
As said before, I don't think it's possible to remove the .approx for mpc320_velocity -> angular velocity conversions, because we will do floating point calculations when converting it into radians/sec, for example. But, I was able to remove the .approx for angular velocity -> mpc320_velocity conversions (because it automatically converts it into int) which is nice!